sway/commands/reload.c

```diff
@@ -22,7 +22,14 @@ static void do_reload(void *data) {
 		list_add(bar_ids, strdup(bar->id));
 	}
 
-	if (!load_main_config(config->current_config_path, true, false)) {
+    const char *path;
```

Comment:
This could be simplified to the following (a ternary would also work, but this looks better due to the line breaks):
```c
const char *path = NULL;
if (config->user_config_path) {
    path = config->current_config_path
}
```